### PR TITLE
O3-1592: Updating the NewVisitPayload to include visitAttributes

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements
- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

#### For changes to apps
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).

#### If applicable
- [x] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

## Summary
This PR updates the NewVisitPayload interface in the visit-resource.tsx to support the visitAttributes.

## Related Issue
https://issues.openmrs.org/browse/O3-1592

## Other
Related to https://github.com/openmrs/openmrs-esm-patient-chart/pull/879

### CODE REVIEW

File: packages/framework/esm-api/src/types/visit-resource.ts [hunk 1]
```diff
@@ -7,6 +7,10 @@ export interface NewVisitPayload {
   startDatetime: Date;
   visitType: string;
   stopDatetime?: Date;
+  attributes?: Array<{
+    attributeType: string;
+    value: string;
```
Comment (ibacher) [inline packages/framework/esm-api/src/types/visit-resource.ts:12]: Maybe `string | number`?

**Review by ibacher — APPROVED** [review]
LGTM aside from the test failure...

File: packages/framework/esm-api/src/types/visit-resource.ts (hunk 1 above)
Comment (vasharma05) [reply packages/framework/esm-api/src/types/visit-resource.ts:12]: image

The UUID is for the `insurance number` in https://oz-kh-dev.mekomsolutions.net

It throws the error: "message": "[attributes on class org.openmrs.Visit => value on class org.openmrs.VisitAttribute => Don't know how to convert from class java.lang.Integer to class java.lang.String]"

Comment (vasharma05) [reply packages/framework/esm-api/src/types/visit-resource.ts:12]: On sending the string:
image

It is stored as a float in the backend.

**Comment by vasharma05** [comment]
Hi @ibacher !
This is weird, the build passed for the PR, but failed when pushed to the code.

**Pull request merged**